### PR TITLE
Move relocate mouse focus

### PR DESCRIPTION
**Pull request by TheNeikos**
Targets #421

### CODE REVIEW

**Comment by neithernut** [comment]
I can ACK this.

File: src/compositor/cursor.c [hunk 1]
```diff
@@ -199,7 +168,7 @@ ws_cursor_set_position(
         ws_cursor_set_image(self, NULL);
         ws_log(&log_ctx, LOG_DEBUG, "Left surface!");
     }
-
+    //
```
Comment (matthiasbeyer) [inline src/compositor/cursor.c]: Is this necessary?

**Comment by matthiasbeyer** [comment]
ACK.

File: src/compositor/cursor.c (hunk 1 above)
Comment (neithernut) [inline src/compositor/cursor.c]: I think this should be removed before merging.

**Comment by TheNeikos** [comment]
Done

**Comment by neithernut** [comment]
ACK.

**Pull request merged**